sound/soc/sof/sof-pci-dev.c

```diff
@@ -381,6 +381,8 @@ static const struct pci_device_id sof_pci_ids[] = {
 #if IS_ENABLED(CONFIG_SND_SOC_SOF_JASPERLAKE)
 	{ PCI_DEVICE(0x8086, 0x38c8),
 		.driver_data = (unsigned long)&jsl_desc},
+	{ PCI_DEVICE(0x8086, 0x4dc8),
+		.driver_data = (unsigned long)&jsl_desc},
```

Comment:
I read in other places that 0x38c8 is ICL-N and 0x4dc8 is JSL-N?
is this all JasperLake really?

Reply:
With jsl+ REV2, the id changed from 0x38c8 to 0x4dc8, but I do not know why it started with 0x38c8 at REV1 PO.

Reply:
Hi, @bardliao @lyakh @libinyang can you review the patch? thanks!!

Reply:
cc @kv2019i for the HDMI part, verified on native port and single Type-C HDMI(via Ho-HO) .